__tests__/url.test.ts

```diff
@@ -3,20 +3,19 @@ import { expect, test } from "vitest"
 import { resolveUrl } from "../lib/helpers/helper.routes"
 
 test("That resolveUrl can return a work url", async () => {
-  const workUrl = resolveUrl({ type: "work", routeParams: { wid: 123 } })
+  const workUrl = resolveUrl({ routeParams: { work: "work", wid: 123 } })
   expect(workUrl).toBe("/work/123")
 })
 
 test("That resolveUrl can return a work url with a manifestation type", async () => {
   const workUrl = resolveUrl({
-    type: "work",
-    routeParams: { wid: 123 },
+    routeParams: { work: "work", wid: 123 },
     queryParams: { audio: "true" },
   })
   expect(workUrl).toBe("/work/123?audio=true")
 })
 
 test("That resolveUrl can return a search url", async () => {
-  const workUrl = resolveUrl({ type: "search", queryParams: { q: "test" } })
+  const workUrl = resolveUrl({ routeParams: { search: "search" }, queryParams: { q: "test" } })
```

Comment:
We talked about taking a look at the `resolveUrl` :) Let's do that on a rainy day.